CMakeLists.txt

```diff
@@ -93,5 +93,5 @@ include(AzureVersion)
 add_subdirectory(sdk/core)
 add_subdirectory(sdk/identity)
 add_subdirectory(sdk/keyvault)
-add_subdirectory(sdk/storage)
+#add_subdirectory(sdk/storage)
```

Comment:
if storage sdk doesn't build, you can just comment it out. You don't have to make all the changes to storage sdk

Reply:
Yes, I just made a few changes as examples, I expect it to be dismissed and storage team would decide how to fix